styleguide/derek/pattern-components/customer-logos/_partials/customer-logo.ejs

```diff
@@ -0,0 +1,7 @@
+<!-- Default description to allow for partial override to demonstrate really long text and equal height -->
+<% if(!img) { var img = "https://0fb555c65f288d09a992-a2085a232a1d2a458a641f48ff12bf08.ssl.cf2.rackcdn.com/LP-assets/All-Tech.png"; } %>
```

Comment:
Files should live in repo, not CDN.

Reply:
I've adjusted all the image paths except the customer logos. They are currently just stub content. I'm not entirely sure they have a place within zoolander since they will be replaced on each LP and they are currently only being used in a test. Does that make sense?